src/apps/dashboard/routes/index.tsx

```diff
@@ -0,0 +1,193 @@
+                onCancel={closeRestartDialog}
+                confirmButtonText={globalize.translate('Restart')}
+                confirmButtonColor='error'
+            />
+            <ConfirmDialog
+                open={isShutdownConfirmDialogOpen}
+                title={globalize.translate('ButtonShutdown')}
+                text={globalize.translate('MessageConfirmShutdown')}
+                onConfirm={onShutdownConfirm}
+                onCancel={closeShutdownDialog}
+                confirmButtonText={globalize.translate('ButtonShutdown')}
+                confirmButtonColor='error'
+            />
+            <Box className='content-primary'>
+                <Grid container spacing={3}>
+                    <Grid size={{ xs: 12, md: 12, xl: 6 }}>
+                        <Stack spacing={3}>
+                            <ServerInfoWidget
+                                systemInfo={systemInfo}
+                                onScanLibrariesClick={onScanLibraries}
+                                onRestartClick={promptRestart}
+                                onShutdownClick={promptShutdown}
+                            />
+                            <ItemCountsWidget counts={itemCounts} />
+                            <RunningTasksWidget tasks={tasks} />
+                            <DevicesWidget devices={devices} />
+                        </Stack>
+                    </Grid>
+                    <Grid size={{ xs: 12, md: 6, lg: 12, xl: 3 }}>
+                        <ActivityLogWidget logs={logs?.Items} />
+                    </Grid>
+                    {isMedium || isExtraLarge ? (
+                        <Grid size={{ md: 6, xl: 3 }}>
+                            <Stack spacing={3}>
+                                <AlertsLogWidget alerts={alerts?.Items} />
+                                <ServerPathWidget systemStorage={systemStorage} />
+                            </Stack>
+                        </Grid>
+                    ) : (
+                        <>
+                            <Grid size={{ xs: 12, lg: 12 }}>
+                                <AlertsLogWidget alerts={alerts?.Items} />
+                            </Grid>
+                            <Grid size={{ xs: 12, lg: 12 }}>
+                                <ServerPathWidget systemStorage={systemStorage} />
+                            </Grid>
+                        </>
+                    )}
+                </Grid>
+
+                <Stack
+                    alignItems='center'
+                    marginTop={4}
+                >
+                    <Link
+                        href='https://jellyfin.org'
+                        target='_blank'
+                        rel='noopener noreferrer'
+                        sx={{
+                            fontWeight: 'bold'
+                        }}
+                    >
+                        Jellyfin
+                    </Link>
+                </Stack>
```

Comment:
I would just remove this... we could add a "widget" for links later, but I don't think many people will find the single link in the footer very helpful